client/src/QuestionList.tsx

```diff
@@ -37,17 +35,33 @@ function QuestionList(props: QuestionListProps) {
 
 const questionStyles = makeStyles<Theme, QuestionProps> ({
     questionWrapper: {
+        boxShadow: 'none!important',
         borderBottom: '1px solid black',
         '& span': {
             fontSize: '1.2rem',
-        }
-    },
-    question: {
-        '& span': {
-            fontWeight: 'bold',
         },
-    },
-    answer: {
+        '&.Mui-expanded': {
+            margin: '0',
+        },
+        '& .MuiAccordionSummary-content': {
+            '& p': {
+                fontWeight: 'bold',
+            },
+        },
+        '& .MuiAccordionSummary-content.Mui-expanded': {
+            margin: 'inherit',
+        },
+        '& .MuiAccordionSummary-root.Mui-expanded': {
+            minHeight: '48px',
+        },
+        '& .MuiButtonBase-root:hover': {
+            backgroundColor: 'rgba(240, 240, 240, 0.5)',
+            
+        },
+        '&.MuiAccordion-root:before': {
+            transition: 'none',
+            backgroundColor: 'transparent',
+        },
```

Comment:
hmmm
i'm thinking there should be a separate class on the summary and some of these would go in a separate root level key of this object that is paired with that
do the &expanded bits apply to it only when expanded or does it look ok when it's not expanded if we put those on the root?
either way, don't spend too much time looking into these, nbd

Reply:
the expanded bits only apply when it's expanded. It's a bit annoying because some of these styles I'm just trying to get rid of what comes with Accordion.

Reply:
I re-organized it, see what you think. It's not as clean as I would like it due to those overrides but I didn't want the text bouncing up and down when people click on questions.

Reply:
yeah it looks fine
originally i was in the train of thought that it'd be more readable to have the logic in JS, but i think that would actually include managing extra state in a onHover action. not ideal
so this might be better anyways
i say we ship now and worry about it later, progress > perfection